### PR TITLE
fix: Parsing timestamptz default string

### PR DESCRIPTION
**Pull request by yevgenypats**
Another edge-case

### CODE REVIEW

File: cqtypes/timestamptz.go [hunk 1]
```diff
@@ -121,6 +121,11 @@ func (dst *Timestamptz) DecodeText(src []byte) error {
 	default:
 		var tim time.Time
 		var err error
+
+		if len(sbuf) > len(defaultStringFormat)+1 && sbuf[len(defaultStringFormat)+1] == 'm' {
+			sbuf = sbuf[:len(defaultStringFormat)]
+		}
```
Comment (hermanschaaf) [inline cqtypes/timestamptz.go:127]: Can we add a test for this case?

**Comment by disq** [comment]
What's the edge case, can you comment in the code? is `2006-01-02 15:04:05.999999999 -0700 MSTm` valid?

File: cqtypes/timestamptz.go (hunk 1 above)
Comment (yevgenypats) [reply cqtypes/timestamptz.go:127]: Added

**Comment by yevgenypats** [comment]
> What's the edge case, can you comment in the code? is `2006-01-02 15:04:05.999999999 -0700 MSTm*` valid?

It's not valid, if you look at `time.String()` they just added manually so when you parse back you have to cut it manually as well

**Review by hermanschaaf — APPROVED** [review]
